### DISCUSSION TITLE
Remove Emote Only messages

### DISCUSSION THREAD

**Discussion by wuast94** [Chatterino/chatterino2#4708] [Ideas]
I would love a feature to Remove that Emote Spam that many channels have. They are unreadable in my opinion. 

here are some examples:

To be Removed:
emote1 emote2 emote1 …
emote1 emote1 emote1 … 

NOT removed:
bla bla emote1

There should be an Option to allow certain emotes, for example:
User1: bla bla bla
User1: Kappa

the kappa emote should be displayed because of context.

**Comment by Felanbird** [comment]
This is very likely possible with a filter that uses regex, but regex that does the same word multiple times is not my area, so I don't have an example.

**Reply by wuast94** [reply]
i dont think so, it must be matched against available emotes. just regexing this would cause: "im doing what im doing" also.

**Reply by pajlada** [reply]
We could add a filter for `contains any text` or something, which would return true if the message contained any text
